packages/stark-ui/src/modules/keyboard-directives/directives/transform-input.directive.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { Component, OnInit } from "@angular/core";
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+import { FormControl, FormsModule, ReactiveFormsModule } from "@angular/forms";
+import { By } from "@angular/platform-browser";
+import { MockStarkLoggingService } from "@nationalbankbelgium/stark-core/testing";
+import { STARK_LOGGING_SERVICE } from "@nationalbankbelgium/stark-core";
+import { StarkTransformInputDirective } from "./transform-input.directive";
+
+/**
+ * @ignore
```

Comment:
There is no need to use `@ignore` here because compodoc does not generate docs for `spec.ts` files. However we do need to add `tslint:disable:completed-docs` comment at the top of this file so that TSLint doesn't complain about missing docs for this file 😉

Reply:
I also removed some redundant documentation.